var/spack/repos/builtin/packages/hip/package.py

```diff
@@ -309,6 +321,7 @@ def get_paths(self):
             }
         else:
             paths = {
+                "hip-path": self.spec,
```

Comment:
Is this one a typo?  I don't think we ever want this to be `self.spec`, right?  Should it be `self.spec.prefix`?

Reply:
@tgamblin Yes this is the bug.

Reply:
@tgamblin Actually that is a second spot that I missed.